config/routes.rb

```diff
@@ -209,7 +210,11 @@
       end
     end
 
-    get '/sign_in' => 'clearance/sessions#new', as: 'sign_in'
+    resource :user, only: [] do
```

Comment:
is `resource only: []` any different from using `namespace` ?

Reply:
Good question, I was following the pattern that currently exists in `routes` for session and I think in this case it makes more sense. `/user` isn't a namespace but a singular resource. Namespacing it would mean that by convention we would expect the controller to exist within a user module.

From the Rails docs
> [2.5 Singular Resources](https://guides.rubyonrails.org/routing.html#singular-resources)
Sometimes, you have a resource that clients always look up without referencing an ID. ..
> [2.6 Controller Namespaces and Routing](https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing)
> You may wish to organize groups of controllers under a namespace. Most commonly, you might group a number of administrative controllers under an Admin:: namespace, and place these controllers under the app/controllers/admin directory. You can route to such a group by using a [namespace](https://api.rubyonrails.org/v7.0.4.2/classes/ActionDispatch/Routing/Mapper/Scoping.html#method-i-namespace) block: